### ISSUE TITLE
Loop Iteration JS and JSON

### ISSUE THREAD

**Issue by jjgreene80** [jjgreene80/James-Greene#1]
Im having an issue looping into the correct array display. Im getting object Object in my table. Any thoughts??

<!DOCTYPE html>
<html>
<head>
    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.1/jquery.min.js"></script>

    <style>
        table, th, td 
        {
            margin:5px 0;
            border:solid 1px #666;
            padding:2px 4px;
            font:15px Tahoma;
        }
        th {
            font-weight:bold;
        }
    </style>
</head>
<body>
    <div id="showData"></div>
</body>
<script>
    $(document).ready(function() {
  $.getJSON("https://itunes.apple.com/us/rss/topalbums/limit=100/json", function(data) {

   var arrItems = [];

   $.each(data.feed.entry, function(index, value) {
    arrItems.push(value);
   });

   var col = [];

   for (var i = 0; i < arrItems.length; i++) {
    for (var key in arrItems[i]) {
     if (col.indexOf(key) === -1) {
      col.push(key);
     }
    }
   }

   var table = document.createElement("table");

   var tr = table.insertRow(-1);

   for (var i = 0; i < col.length; i++) {
    var th = document.createElement("th");
    th.innerHTML = col[i];
    tr.appendChild(th);
   }

   for (var i = 0; i < arrItems.length; i++) {

    tr = table.insertRow(-1);

    for (var j = 0; j < col.length; j++) {
     var tabCell = tr.insertCell(-1);
     tabCell.innerHTML = arrItems[i][col[j]];
    }
   }

   var divContainer = document.getElementById("showData");
   divContainer.innerHTML = "";
   divContainer.appendChild(table);
  });
 });
</script>
</html>
